kalite/version.py

```diff
@@ -7,7 +7,7 @@
 # Must also be of the form N.N.N for internal use, where N is a non-negative integer
 MAJOR_VERSION = "0"
 MINOR_VERSION = "14"
-PATCH_VERSION = "0"
+PATCH_VERSION = "alpha3"
```

Comment:
This will get automatically "cleaned up" by setuptools, which I discovered when running the command `setup.py sdist --static`. "0.14.alpha3" gets turned into "0.14a3" (or something like that). Basically you can get into trouble if you expect the version to look like the former but it looks like the latter.